src/field/pokemon.ts

```diff
@@ -4573,20 +4572,22 @@ export class EnemyPokemon extends Pokemon {
     if (movePool.length) {
       // If there's only 1 move in the move pool, use it.
       if (movePool.length === 1) {
-        return { moveId: movePool[0].moveId, targets: this.getNextTargets(movePool[0].moveId) };
+        const move = movePool[0].getMove();
+        return { move, targets: this.getNextTargets(movePool[0].moveId), type: this.getMoveType(move) };
```

Comment:
```suggestion
        return { move, targets: this.getNextTargets(move.id), type: this.getMoveType(move) };
```
it's the same, but more consistent style wise imo